### PR TITLE
Update uuid

### PR DESCRIPTION
**Pull request by Eijebong**
---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/22098)

### CODE REVIEW

File: Cargo.lock [hunk 1]
```diff
@@ -3046,7 +3046,7 @@ dependencies = [
  "lazy_static 1.0.0 (registry+https://github.com/rust-lang/crates.io-index)",
  "libc 0.2.42 (registry+https://github.com/rust-lang/crates.io-index)",
  "num_cpus 1.4.0 (registry+https://github.com/rust-lang/crates.io-index)",
- "rand 0.4.2 (registry+https://github.com/rust-lang/crates.io-index)",
+ "rand 0.3.22 (registry+https://github.com/rust-lang/crates.io-index)",
```
Comment (atouchet) [inline Cargo.lock]: Why is rand being downgraded here?

**Comment by atouchet** [comment]
ipc-channel 0.11.1 is now published.

File: Cargo.lock (hunk 1 above)
Comment (Eijebong) [reply Cargo.lock]: cargo doing weird stuff... It's not the first time this happens...

**Comment by Eijebong** [comment]
>   │ Failed to start unsandboxed child process!: Os { code: 12, kind: Other, message: "Cannot allocate memory" } (thread Constellation, at libcore/result.rs:1009)

**Comment by jdm** [comment]
We can retry this when it's rebased.

**Comment by Eijebong** [comment]
Sure, waiting for the rustc serialize one to land to avoid rebasing twice

**Comment by Eijebong** [comment]
Rebased

**Comment by Eijebong** [comment]
`OSError: [Errno 12] Cannot allocate memory`

**Comment by Eijebong** [comment]
Rebased

**Comment by Eijebong** [comment]
`{"status": "FAIL", "group": "default", "message": "/css/CSS2/fonts/font-148.xht 49ebf872ef928daadc64184878066c1783e136c5\n/css/CSS2/fonts/font-148-ref.xht 4e6ec2a4e7504e5c4bb739eb4fad793251d3180f\nTesting 49ebf872ef928daadc64184878066c1783e136c5 == 4e6ec2a4e7504e5c4bb739eb4fad793251d3180f", "stack": null, "subtest": null, "test": "/css/CSS2/fonts/font-148.xht", "line": 5121, "action": "test_result", "expected": "PASS"}`

**Comment by jdm** [comment]
@bors-servo retry
* #22062

**Pull request merged**